pkg/stores/sqlpartition/queryparser/labels.go

```diff
@@ -0,0 +1,164 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+This file is derived from
+https://github.com/kubernetes/apimachinery/blob/master/pkg/labels/labels.go
```

Comment:
Looking at the diff between the file pointed by this link and the file in this PR I see they have:

 - a `FormatLabels` func we don't
 - more validation in `ConvertSelectorToLabelsMap` (but we do not use that func)

Is this correct? If so I would suggest:

```suggestion
This file is derived from
https://github.com/kubernetes/apimachinery/blob/90df4d1d2d40ea9b3a522bec6e3577237358de00/pkg/labels/labels.go

 - FormatLabels was dropped
 - validateLabelKey calls from ConvertSelectorToLabelsMap were dropped
```

That will help if ever this needs bumping

Reply:
ok